api/src/middlewares/services/UserService/personalisation/personalisationBuilder.userConfirmation.ts

```diff
@@ -26,47 +38,30 @@ export function buildUserConfirmationPersonalisation(answers: AnswersHashMap, me
   return {
     firstName: answers.firstName,
     post: getPost(country, post),
-    docsList,
     country,
     bookingLink: additionalContext.bookingLink,
     localRequirements: additionalContext.localRequirements,
     civilPartnership: additionalContext.civilPartnership,
     reference: metadata.reference,
     confirmationDelay: additionalContext.confirmationDelay ?? "2 weeks",
     notPaid: !isSuccessfulPayment,
+    ...additionalPersonalisations,
+    additionalDocs: additionalContext.additionalDocs ?? "",
   };
 }
 
-export function buildUserConfirmationDocsList(fields: AnswersHashMap, type?: FormType) {
-  if (!fields) {
-    throw new ApplicationError("WEBHOOK", "VALIDATION", 500, "Fields are empty");
-  }
-
-  const docsList = ["your UK passport", "your partner’s passport or national identity card"];
-
-  // for cni applications, proof of address is only required if the user lives in the country. For affirmation applications, proof of address is always required
-  if (type === "affirmation" || (type === "cni" && fields.livesInCountry)) {
-    docsList.splice(1, 0, "proof of address – you must use your residence permit if the country you live in issues these");
-  }
-
-  // for cnis, the user needs to provide proof they have stayed in the country for 3 days. For contextual reasons, this should appear below the proof of address doc
-  if (type === "cni" && !fields.livesInCountry) {
-    docsList.splice(1, 0, "proof you’ve been staying in the country for 3 whole days before your appointment – if this is not shown on your proof of address");
-  }
-
-  // for affirmations, users need to provide their birth certificate. For contextual reasons, this should appear next to the user's passport
-  if (type === "affirmation") {
-    docsList.splice(1, 0, "your birth certificate");
-  }
+export function getAffirmationPersonalisations(fields: AnswersHashMap) {
```

Comment:
rename/move `personalisationBuilder.userConfirmation` to `/personalisationBuilder/userConfirmation/index.ts`, and then move `personalisationTypeMap` and related functions to 

`/personalisationBuilder/userConfirmation/getAdditionalPersonalisations.ts`

rename `personalisationTypeMap` to just `additionalPersonalisations`.